src/client/Microsoft.Identity.Client/BearerAuthenticationScheme.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Microsoft.Identity.Client.Cache.Items;
+
+namespace Microsoft.Identity.Client
+{
+    internal class BearerAuthenticationScheme : IAuthenticationScheme
+    {
+        public string AuthorizationHeaderPrefix => "Bearer";
```

Comment:
In the earlier code (CreateAuthorizationHeader) you're doing "Bearer" + " " + AccessToken.  So do we need a method on the IAuthenticationScheme for GetAuthorizationHeader() that does the proper formatting here?  This will just get you the prefix which isn't enough to actually create the header in AuthenticationResult.CreateAuthorizationHeader().

Reply:
We don't need this abtraction for Bearer, POP or SSH. For SSH calling `CreateAuthorizationHeader()` would just throw.